.gitignore

```diff
@@ -20,6 +20,4 @@ config.local.*
 setup.py~
 .idea/
 htmlcov/
-docs/source/.static/
-*.png
```

Comment:
After removing this line, we will need to add a more specific version to deal with the .png files generated by unit tests. I'll take care of that.

Reply:
I'll also need to add the .png file to git... another commit, then